From aee0be27a8e8753ac68adc035f098ccc7a9e3678 Mon Sep 17 00:00:00 2001
From: Sebastian Berg <sebastianb@nvidia.com>
Date: Tue, 27 Aug 2024 17:39:20 -0700
Subject: [PATCH 01/12] MAINT: fix Python 3.13 compatibility

---
 pytest_doctestplus/plugin.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/pytest_doctestplus/plugin.py b/pytest_doctestplus/plugin.py
index f9c3a0c..5538c53 100644
--- a/pytest_doctestplus/plugin.py
+++ b/pytest_doctestplus/plugin.py
@@ -900,7 +900,8 @@ def write_modified_file(fname, new_fname, changes):
         lineno = change["test_lineno"] + change["example_lineno"]
         lineno += change["source"].count("\n")
 
-        indentation = " " * change["nindent"]
+        indentation = len(text[lineno-1]) - len(text[lineno-1].lstrip())
+        indentation = text[lineno-1][:indentation]
         want = indent(change["want"], indentation, lambda x: True)
         # Replace fully blank lines with the required `<BLANKLINE>`
         # (May need to do this also if line contains only whitespace)
-- 
2.45.2

