diff -ur python-casacore-3.1.1/setup.py python-casacore-3.1.1_ed/setup.py
--- python-casacore-3.1.1/setup.py	2019-06-14 10:01:01.000000000 +0300
+++ python-casacore-3.1.1_ed/setup.py	2019-10-18 15:24:02.721012259 +0300
@@ -50,7 +50,7 @@
     # Append default search path (not a complete list)
     lib_dirs = user_lib_dirs + [os.path.join(sys.prefix, 'lib'),
                               '/usr/local/lib',
-                              '/usr/lib',
+                              '/usr/lib64',
                               '/usr/lib/x86_64-linux-gnu']
 
     if 'LD_LIBRARY_PATH' in os.environ:
@@ -67,8 +67,10 @@
 def find_boost():
     """Find the name of the boost-python library. Returns None if none is found."""
     short_version = "{}{}".format(sys.version_info[0], sys.version_info[1])
+    short_version1 = "{}.{}".format(sys.version_info[0], sys.version_info[1])
     boostlibnames = ['boost_python-py' + short_version,
                      'boost_python' + short_version,
+                     'boost_python-' + short_version1,
                      'boost_python',
                      ]
     # The -mt (multithread) extension is used on macOS but not Linux.
@@ -194,7 +196,7 @@
 setup(name='python-casacore',
       version=__version__,
       description='A wrapper around CASACORE, the radio astronomy library',
-      install_requires=['numpy', 'argparse', 'future', 'six'],
+      install_requires=['numpy', 'future', 'six'],
       author='Gijs Molenaar',
       author_email='gijs@pythonic.nl',
       url='https://github.com/casacore/python-casacore',
