diff -ur arviz-0.7.0.orig/arviz/tests/base_tests/test_plots_matplotlib.py arviz-0.7.0/arviz/tests/base_tests/test_plots_matplotlib.py
--- arviz-0.7.0.orig/arviz/tests/base_tests/test_plots_matplotlib.py	2020-03-03 01:04:25.000000000 +0300
+++ arviz-0.7.0/arviz/tests/base_tests/test_plots_matplotlib.py	2020-05-21 15:53:13.054294324 +0300
@@ -152,6 +152,7 @@
     assert axes.shape
 
 
+@pytest.mark.skip(reason="needs internet access")
 @pytest.mark.parametrize("compact", [True, False],)
 @pytest.mark.parametrize("combined", [True, False],)
 def test_plot_trace_legend(compact, combined):
