--- a/duplicity/backends/lftpbackend.py	2017-08-06 19:25:09.000000000 +0300
+++ b/duplicity/backends/lftpbackend.py	2019-04-03 17:57:00.102709297 +0300
@@ -128,9 +128,12 @@
         if log.getverbosity() >= log.DEBUG:
             os.write(self.tempfile, "debug\n")
         if self.parsed_url.scheme == 'ftpes':
-            os.write(self.tempfile, "open %s %s\n" % (self.authflag, self.url_string.replace('ftpes', 'ftp')))
+            os.write(self.tempfile, "set cmd:default-protocol %s\n" % 'ftps')
+            # os.write(self.tempfile, "open %s %s\n" % (self.authflag, self.url_string.replace('ftpes', 'ftp')))
         else:
-            os.write(self.tempfile, "open %s %s\n" % (self.authflag, self.url_string))
+            os.write(self.tempfile, "set cmd:default-protocol %s\n" % self.scheme)
+            # os.write(self.tempfile, "open %s %s\n" % (self.authflag, self.url_string))
+        os.write(self.tempfile, "open %s %s\n" % (self.authflag, self.parsed_url.hostname))
         os.close(self.tempfile)
         # print settings in debug mode
         if log.getverbosity() >= log.DEBUG:
