diff -ur PDL-Fit-Levmar-0.0098/pdlperlfunc.c PDL-Fit-Levmar-0.0098_ed/pdlperlfunc.c
--- PDL-Fit-Levmar-0.0098/pdlperlfunc.c	2015-11-29 16:48:11.000000000 +0300
+++ PDL-Fit-Levmar-0.0098_ed/pdlperlfunc.c	2016-11-01 15:49:47.000000000 +0300
@@ -21,10 +21,10 @@
 // following line useless, i guess
 // typedef void (*levmarfunc)( double *, double *, int, int, void * ) ;
 
-typedef void (*DelMagic)(pdl *, int param);
-static void delete_levmar_pdls(pdl* p, int param);
+typedef void (*DelMagic)(pdl *, size_t param);
+static void delete_levmar_pdls(pdl* p, size_t param);
 
-static void default_magic(pdl *p, int pa) { p->data = 0; }
+static void default_magic(pdl *p, size_t pa) { p->data = 0; }
 static pdl* pdl_wrap(void *data, int datatype, PDL_Indx dims[],
 		     int ndims, DelMagic delete_magic, int delparam);
 static SV *getref_pdl(pdl *it);
@@ -178,7 +178,7 @@
 }
 
 // Don't free data, it is allocated inside liblevmar
-static void delete_levmar_pdls(pdl* p, int param)
+static void delete_levmar_pdls(pdl* p, size_t param)
 {
   if (p->data) {
     // free(p->data); 
