diff -ur Theano-PyMC-1.0.13.orig/setup.py Theano-PyMC-1.0.13/setup.py
--- Theano-PyMC-1.0.13.orig/setup.py	2020-12-15 04:03:57.000000000 +0300
+++ Theano-PyMC-1.0.13/setup.py	2020-12-16 02:36:39.694352378 +0300
@@ -56,7 +56,7 @@
         url=URL,
         license=LICENSE,
         platforms=PLATFORMS,
-        packages=find_packages(exclude=["tests.*"]),
+        packages=find_packages(exclude=["tests*"]),
         install_requires=["numpy>=1.9.1", "scipy>=0.14"],
         package_data={
             "": [
