diff -ur Theano-PyMC-1.0.11.orig/setup.py Theano-PyMC-1.0.11/setup.py
--- Theano-PyMC-1.0.11.orig/setup.py	2020-11-15 06:35:27.000000000 +0300
+++ Theano-PyMC-1.0.11/setup.py	2020-12-07 02:05:14.204699113 +0300
@@ -106,7 +106,7 @@
         url=URL,
         license=LICENSE,
         platforms=PLATFORMS,
-        packages=find_packages(),
+        packages=find_packages(exclude=['tests*']),
         cmdclass=versioneer.get_cmdclass(),
         install_requires=["numpy>=1.9.1", "scipy>=0.14"],
         package_data={
