--- a/src/dataStore/RouteEntity.cpp	2025-07-08 14:05:47.000000000 +0300
+++ b/src/dataStore/RouteEntity.cpp	2025-07-22 17:34:43.410738481 +0300
@@ -812,6 +812,7 @@
             Rules.push_back(std::make_shared<RouteRule>(*item));
         }
         fn = QString(other.fn);
+        defaultOutboundID = other.defaultOutboundID;
 
         _add(new configItem("id", &id, itemType::integer));
         _add(new configItem("name", &name, itemType::string));
