diff -ur wcslib-7.7.orig/Fortran/GNUmakefile wcslib-7.7/Fortran/GNUmakefile
--- wcslib-7.7.orig/Fortran/GNUmakefile	2021-07-12 09:36:49.000000000 +0300
+++ wcslib-7.7/Fortran/GNUmakefile	2021-08-30 02:32:55.132647773 +0300
@@ -361,7 +361,7 @@
 	-@ echo ''
 	   $(MAKE) -C ../pgsbox lib
 
-tofits : ../C/test/tofits.c
+tofits : ../utils/tofits.c
 	   $(CC) $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) -o $@ $<
 
 ../C/wcstab.fits : ;
